types/ramda/ramda-tests.ts

```diff
@@ -925,8 +925,8 @@ interface Obj {
 };
 
 () => {
-    R.flatten([1, 2, [3, 4], 5, [6, [7, 8, [9, [10, 11], 12]]]]);
-    // => [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+    R.flatten([[1, 2], [2, 3], [3, 4]]); // => [1, 2, 3, 4, 5, 6]
```

Comment:
wrong
`R.flatten([[1, 2], [2, 3], [3, 4]]);` will give you `[1, 2, 2, 3, 3, 4]`